xarray/datatree_/docs/source/quick-overview.rst

```diff
@@ -0,0 +1,84 @@
+.. currentmodule:: datatree
```

Comment:
open to ideas about how to handle this content. A stand along page on datatree may make sense for a while.

Reply:
This content was deliberately written so that it could be eventually slotted underneath the [`Datasets` section on the Quick overview page](https://docs.xarray.dev/en/stable/getting-started-guide/quick-overview.html#datasets).